docs/content/templates/functions.md

```diff
@@ -662,6 +662,15 @@ e.g.
 * `{{slicestr "BatMan" 3}}` → "Man"
 * `{{slicestr "BatMan" 0 3}}` → "Bat"
 
+### truncate
+
+Truncate a text to a max length without cutting words or HTML tags in half. Since go templates are HTML aware, truncate will handle normal strings vs HTML strings intelligently.
```

Comment:
I would say:

Truncate a text to a max length without cutting words or leaving unclosed HTML tags.  Since Go templates are HTML-aware, `truncate` will handle normal strings vs HTML strings intelligently.  It's important to note that if you have a raw string that contains HTML tags that you want treated as HTML, you will need to convert the string to HTML using the `safeHTML` template function before sending the value to `truncate`; otherwise, the HTML tags will be escaped by `truncate`.

```
`{{ "<em>Keep my HTML</em>" | safeHTML | truncate 10 }}` → `<em>Keep my …</em>`
```